modules/cnf-configuring-huge-pages.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//CNF-78
+// * scalability_and_performance/cnf-configuring-huge-pages.adoc
+
+[id="cnf-configuring-huge-pages_{context}"]
+= Configuring huge pages
+
+Nodes must pre-allocate huge pages used in an {product-title} cluster. Use the
+Performance Addon Operator to allocate huge pages on a specific node.
+
+{product-title} provides a method for creating and allocating huge pages.
+Performance Addon Operator provides an easier method for doing this using the PerformanceProfile, as described in
+xref:../modules/configuring-huge-pages.adoc[Configuring huge pages].
+
+For example, in the `hugepages` `pages` section of the PerformanceProfile,
+you can specify multiple blocks of `size`, `count`, and, optionally, `node`, as shown here:
+
+----
+hugepages:
+   defaultHugepagesSize: "1Gi"
+   pages:
+   - size:  "1Gi"
+     count:  4
+     node:  0 <1>
+----
+
+<1> `node` is the NUMA node in which the huge pages are allocated. If you omit `node`, the pages are evenly spread across all NUMA nodes.
+
+[NOTE]
+====
+Wait for the relevant machineconfigpool status that indicates it has finished the update.
+====
+
+These are the only configuration steps you need to do to allocate huge pages.
+For more information about how to consume hugepages from your containers
+see https://docs.openshift.com/container-platform/4.4/scalability_and_performance/what-huge-pages-do-and-how-they-are-consumed-by-apps.html#how-huge-pages-are-consumed-by-apps_huge-pages[How huge pages are consumed by apps].
+
+To verify the configuration, see the `/proc/meminfo` file on the node.
+For information on using `oc debug node` see Step 6 here: https://docs.openshift.com/container-platform/4.4/nodes/nodes/nodes-nodes-working.html#nodes-nodes-kernel-arguments_nodes-nodes-working[Adding kernel arguments to Nodes].
```

Comment:
We have to drop this link. Maybe you can manually add step 6 here in line?

Reply:
Manually added Step 6.